microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -251,6 +251,33 @@ DevTools shows you a breakdown of memory allocation by function.  The default vi
 ![Allocation sampling.](../media/memory-problems-glitch-example-05-memory-allocation-sampling-heavy-bottom-up.msft.png)
 
 
+<!-- ====================================================================== -->
+## Reduce garbage with additional settings for Allocation sampling
```

Comment:
```suggestion
## Reduce garbage with additional settings for allocation sampling
```
Judging when upper vs lowercase A:
When the context is closely focused on the UI, can lean toward "Allocation sampling".  The word 'allocation' is lowercased in the UI in mid-sentences, it's only cap'd when first word of label.